slackgpt/Queue.py

```diff
@@ -22,6 +22,15 @@ def pop(self):
         """
         return self.__queue.pop(0)
 
+    def insert(self, index, item):
```

Comment:
For what is this needed? thats some special queue behavior :D

Reply:
for when i need to put something at the front of the queue -> answering of question fails